docs/api/search-autocomplete-service-resource.md

```diff
@@ -50,6 +50,7 @@ skip        | URL    | integer | no       | The number of results to skip, for p
 take        | URL    | integer | no       | The number of results to return, for pagination
 prerelease  | URL    | boolean | no       | `true` or `false` determining whether to include [pre-release packages](../create-packages/prerelease-packages.md)
 semVerLevel | URL    | string  | no       | A SemVer 1.0.0 version string 
+packageType | URL    | string  | no       | The package type to use to filter packages
```

Comment:
Could you mention `SearchAutocompleteService/3.5.0` and `SearchQueryService/3.5.0` similar to how we mentioned 3.6.0 for for registration?
https://docs.microsoft.com/en-us/nuget/api/registration-base-url-resource#versioning

Basically we add a row to the table and have a dedicated section about new version. We should emphasize that it's a) backwards compatible for `SearchAutocompleteService` and introduces the `packageType` parameter and `packageTypes` response property (just for search, not autocomplete of course)